manifests/slave.pp

```diff
@@ -150,7 +150,7 @@
     ensure => 'file',
     mode   => '0755',
     owner  => 'root',
-    group  => 'root',
+    group  => '0',
```

Comment:
Probably needs to be in the params.pp, which is the right place for this kind of thing.